modules/hub/pages/hub-api-installation.adoc

```diff
@@ -0,0 +1,21 @@
+[[hub-api-installation]]
+= Hub XMRPC API Installation
+
+
+Install the package hub-xmlrpc-api, available from SUSE Manager 4.1 and Uyuni repos, either on the Hub itself or on a host that has access to all Servers' XMLRPC APIs.
+
+Configuration of `hub-xmlrpc-api` is specified in a JSON file like the following:
+[source,json]
+----
+{
+   "type": "json",
+    "hub": {
+       "manager_api_url": "http://localhost/rpc/api"
+   },
+    "connect_timeout": 10,
```

Comment:
I know it's quite intuitive, but shouldn't we add a small explanation of the timeouts anyway?

Reply:
Yeah, we should.  :+1: